ls1intum/Artemis
# `Assessment`: Only notify students at assessment due date if they actually received an assessment

### Checklist
#### General


#### Server

### Motivation, Context and Description


Fix a bug where all students received a notification at the assessment due date, even if their assessment was not finished.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 2 Students
- 1 Programming Exercise with manual assessment

1. Log in to Artemis
2. Participate with both students in the exercise
3. Assess one before the **assessment** due date
4. Check that only the student with an assessment got the notification at the assessment due date
5. Assess the other submission
6. Check that the other student now has a notification as well

### Review Progress






#### Code Review
#### Manual Tests

### Test Coverage





### Screenshots
